src/content/pages/sponsor.mdx

```diff
@@ -41,7 +41,7 @@ accessible to most organisations keen to help the community.
 <div style={{textAlign: "left", marginBottom: "30px"}}>
```

Comment:
could be nicer to use tailwind's space-y class here, instead of using a class on the button: https://tailwindcss.com/docs/space

what do you think? 😊